packages/widget/src/components/SendToWallet/SendToWalletButton.tsx

```diff
@@ -103,14 +98,23 @@ export const SendToWalletButton: React.FC<CardProps> = (props) => {
     setSelectedBookmark();
   };
 
-  const openCollapse = disabledToAddress
-    ? showSendToWallet || (showInstantly && !showSendToWalletDirty)
-    : showSendToWallet || showInstantly;
+  // The collapse opens instantly on first page load/component mount when there is an address to display
+  // After which it needs an animated transition for open and closing.
+  // collapseTransitionTime is used specify the tranistion time for opening and closing
+  const [collapseTransitionTime, setCollapseTransitionTime] = useState(0);
+
+  useEffect(() => {
+    // this should fire after the collapse have been opened instantly
+    setCollapseTransitionTime(225);
```

Comment:
Do you think we could change this to use useRef to avoid rerender? Not a fan of updating state after mount if effect 😅🙏